DigitalLearningSolutions.Data/Services/TrackerService.cs

```diff
@@ -14,11 +16,13 @@ public class TrackerService : ITrackerService
     {
         private readonly ILogger<TrackerService> logger;
         private readonly ITrackerActionService trackerActionService;
+        private readonly JsonSerializerSettings settings;
 
         public TrackerService(ILogger<TrackerService> logger, ITrackerActionService trackerActionService)
         {
             this.logger = logger;
             this.trackerActionService = trackerActionService;
+            settings = new JsonSerializerSettings { ContractResolver = new LowercaseContractResolver() };

```

Comment:
Slightly in favour of a field initializer rather than doing this in the constructor because it doesn't depend on any services.